src/lib/address/address.test.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  btcAssetId,
+  btcChainId,
+  ethAssetId,
+  ethChainId,
+  ltcAssetId,
+  ltcChainId,
+} from '@shapeshiftoss/caip'
+
+import { parseMaybeUrlByChainId } from './address'
+
+describe('lib/address', () => {
+  describe('parseMaybeUrlByChainId', () => {
+    it('should not parse EIP-681 URL for ENS domain', () => {
+      const input = {
+        assetId: ethAssetId,
+        chainId: ethChainId,
+        value: 'vitalik.eth',
+      }
+
+      const expectedOutput = {
+        assetId: ethAssetId,
+        chainId: ethChainId,
+        value: 'vitalik.eth',
+      }
+
+      expect(parseMaybeUrlByChainId(input)).toEqual(expectedOutput)
+    })
+
+    it('should parse address from EIP-681 URL without parameters', () => {
+      const input = {
+        assetId: ethAssetId,
+        chainId: ethChainId,
+        value: 'ethereum:0x1234DEADBEEF5678ABCD1234DEADBEEF5678ABCD',
+      }
+
+      const expectedOutput = {
+        assetId: ethAssetId,
+        chainId: ethChainId,
+        value: '0x1234DEADBEEF5678ABCD1234DEADBEEF5678ABCD',
+      }
+
+      expect(parseMaybeUrlByChainId(input)).toEqual(expectedOutput)
+    })
+
+    it('should not parse EIP-681 URL with parameters', () => {
+      const input = {
+        assetId: ethAssetId,
+        chainId: ethChainId,
+        value:
+          'ethereum:0x1234DEADBEEF5678ABCD1234DEADBEEF5678ABCD?value=2.014e18&gas=10&gasLimit=21000&gasPrice=50',
```

Comment:
getting some bizarre behavior pasting this address into the send modal, a weirdly rendered x character (copy pasted from in the input in the screenshot) ->`x`

<img width="507" alt="Screenshot 2023-04-11 at 11 12 34 AM" src="https://user-images.githubusercontent.com/88504456/231239063-757f5213-e449-4406-8336-3d96df586d77.png">

Reply:
LLM rug?

Reply:
Now that's correct, it's just the new font, here it is pasted here: `ethereum:0x1234DEADBEEF5678ABCD1234DEADBEEF5678ABCD?value=2.014e18&gas=10&gasLimit=21000&gasPrice=50`

The behavior is actually correct and so is the test. We should never parse EIP-681 with params as the test suggests. If you parse the l.34 one e.g `ethereum:0x1234DEADBEEF5678ABCD1234DEADBEEF5678ABCD` everything will be 🛍️ 
<img width="530" alt="image" src="https://user-images.githubusercontent.com/17035424/231290852-704260f8-12d9-4fc4-9fb8-4aed9aae60e2.png">

Reply:
hahaha fucking font rug. i checked with `charCodeAt` and it evaluated the same, so in that case, LGTM.